### PR TITLE
Update paperless-ng to 1.5.0

### LINKED ISSUE #11: Still maintained?
Hello,

@TheBestMoshe do you still maintain that repo? I don't see activity since February. It is just a question. I don't want to push you or anything else.

Thanks, Thomas

### PR DESCRIPTION
**Pull request by lociii**
Update release by release.

### CODE REVIEW

File: paperless-ng/redis.conf [hunk 1]
```diff
@@ -300,7 +300,7 @@ always-show-logo yes
 #
 #   It is also possible to remove all the previously configured save
 #   points by adding a save directive with a single empty string argument
-#   like in the following example:
+#   like in the following example:l
```
Comment (TheBestMoshe) [inline paperless-ng/redis.conf]: @lociii Is this a mistake?

**Review by TheBestMoshe — COMMENTED** [review]
@lociii Thanks for this PR!

File: paperless-ng/redis.conf (hunk 1 above)
Comment (TheBestMoshe) [reply paperless-ng/redis.conf]: Once you remove this, I can merge the PR. Everything else looks good.

**Comment by TheBestMoshe** [comment]
BTW, regarding the devcontainer configuration, I am happy to add it to the repo if you want. Could you just open a separate PR for it?

File: paperless-ng/redis.conf (hunk 1 above)
Comment (lociii) [reply paperless-ng/redis.conf]: Thanks for pointing this out.
I'm on holiday and away from home this week and will only be able to fix it next week.

**Comment by lociii** [comment]
I'm super happy with the gitpod setup and see no need for a devcontainer anymore.

**Comment by lociii** [comment]
@TheBestMoshe unintentional change in redis.conf reverted. Ready to merge now.

File: paperless-ng/redis.conf (hunk 1 above)
Comment (lociii) [reply paperless-ng/redis.conf]: Done.

**Comment by TheBestMoshe** [comment]
Thanks @lociii

**Pull request merged**